etc/sabakan-state-setter.yml

```
@@ -3,29 +3,6 @@ machine-types:
     grace-period: 5s
     metrics:
       - name: hw_processor_status_health
```

Ah, how about this?

```suggestion
      - name: processor_status_health
```

c.f. https://github.com/cybozu-go/setup-hw/blob/master/redfish/rules/qemu.yml#L8